styles/main.css

```diff
@@ -76,10 +84,11 @@ figure img{
 
 .width-wrap{
 	box-sizing: border-box;
-	border: 1px solid rgba(0,0,0.3);
+	border: 1px solid rgba(0,0,0,0.3);
+	border-top: none;
 	margin:0 auto;
     height: 100%;
-    width: 1024px;
+    max-width: 1024px;
```

Comment:
Setting max-with instead of width allows the element to become smaller if the browser window shrinks, but still prevents it from getting wider than 1024px